pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions.go

```diff
@@ -130,6 +131,15 @@ func (pl *VolumeRestrictions) Filter(ctx context.Context, _ *framework.CycleStat
 	return nil
 }
 
+// EventsToRegister returns the possible events that may make a Pod
+// failed by this plugin schedulable.
+func (pl *VolumeRestrictions) EventsToRegister() []framework.ClusterEvent {
+	return []framework.ClusterEvent{
+		{Resource: framework.Pod, ActionType: framework.Delete | framework.Update},
```

Comment:
lets add a comment on each explaining how they impact schedulability of pods.

Reply:
@cofyc can you help advise?

Reply:
see example: https://github.com/kubernetes/kubernetes/pull/100003

Reply:
ok

Reply:
Thanks for your suggestion. I've added inline comments with the latest push.